content/german/components/sound-sensor.md

```diff
@@ -0,0 +1,68 @@
+---
+
+title: Sound Sensor
+tags: ["gpio"]
+--------------
```

Comment:
```suggestion
---
title: Sound Sensor
tags: ["gpio"]
---
```

Currently the front matter is broken and there's a weird line after the heading. This should fix that.